src/components/Buttons.js

```diff
@@ -12,23 +12,23 @@ const Buttons = ({ clear, equalityHandler }) => {
   return(
     <Box>
       <Button text={'AC'} color={'darkGray'} onClick={clear}/>
-      <Button text={'+/-'} color={'darkGray'}/>
-      <Button text={'%'} color={'darkGray'}/>
-      <Button text={'÷'} color={'orange'}/>
-      <Button text={'7'}/>
-      <Button text={'8'}/>
-      <Button text={'9'}/>
-      <Button text={'x'} color={'orange'}/>
-      <Button text={'4'}/>
-      <Button text={'5'}/>
-      <Button text={'6'}/>
-      <Button text={'-'} color={'orange'}/>
-      <Button text={'1'}/>
-      <Button text={'2'}/>
-      <Button text={'3'}/>
-      <Button text={'+'} color={'orange'}/>
-      <Button text={'0'} size={'big'}/>
-      <Button text={','}/>
+      <Button text={'+/-'} color={'darkGray'} onClick={reverseSign}/>
```

Comment:
Let's refactor this code a little bit. You can create a config for all these buttons with all the props you need for them. Then you can pass the same function for all the buttons in order to handle the `onClick` event and manage what happens based on the text of the clicked button inside that function (you can use a `switch case` for this).